tests/kolainst/destructive/staged-deploy.sh

```diff
@@ -98,6 +98,11 @@ EOF
     syncfs=$(journalctl -b -1 -u ostree-finalize-staged --grep='Completed syncfs.*for system repo' | wc -l)
     assert_streq "$syncfs" 2
 
+    # And verify there were no failures to unmount /etc or /sysroot
+    journalctl -b -1 -u sysroot.mount -u etc.mount >previous-journal.txt
```

Comment:
Minor: looks like it accepts globs, so `-u '*.mount'` also works and catches more.